packages/react-core/src/components/Panel/Panel.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Panel/panel';
+import { css } from '@patternfly/react-styles';
+
+export interface PanelProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside Panel */
+  children?: React.ReactNode;
+  /** Class to add to outer div */
+  className?: string;
+  /** Flag to add raised styling to the panel */
+  isRaised?: boolean;
+  /** Flag to add bordered styling to the panel */
+  isBordered?: boolean;
```

Comment:
fwiw, I don't think you would ever have `isRaised && isBordered` - it would be one or the other. Assuming that's the case, would you keep these props as separate or have a prop that takes raised, bordered, etc as values?

Reply:
I can definitely see the appeal of that approach (assuming raised and bordered will indeed never be used together), especially if it is anticipated that additional exclusive styles might be added in the future. I'm not totally sure that getting rid of one prop is worth adding the additional prop evaluation required for that approach though.

I'm happy to refactor to that approach if yall think it's the best way forward and we confirm raised and bordered will be exclusive.

Reply:
I think combining the two would help prevent consumer error.  I don't know what a good name for the prop would be tough.

Reply:
@mcarrano can you confirm whether the raised and bordered styles would ever be used together? I'm assuming it's like a card where you use one or the other, but never together.

Reply:
Yes, that's correct. I can't think of a reason to use these together.